### PR TITLE
My_Solution

### PR DESCRIPTION
**Pull request by neagarr**
@Abnormaltype 
I would like to ask: why "person_list[i].wife = pair_person" and "person_list[i].husband = pair_person" have the warning "Unresolved attribute reference 'wife' for class 'Person'" in Pycharm?  
According to the course materials, we can complete adding new instance attributes by using instance.attribute_name = value

### CODE REVIEW

**Review by Abnormaltype — COMMENTED** [review]
because you don't have annotation for these list)

**Comment by neagarr** [comment]
So, where I should make the annotation? Should I?

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
```
Comment (Abnormaltype) [inline app/main.py:11]:
```suggestion
    person_list: list[Person] = []
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by neagarr** [comment]
I added list[Person], but nothing changed. Now :
1. "person_list: list[Person] = [] has the warning "Builtin 'list' cannot be parameterized directly"
2. "person_list[i].wife = pair_person" and "person_list[i].husband = pair_person" have the warning "Unresolved attribute reference 'wife' for class 'Person'"

But everything works)

**Review by IvanRamyk — APPROVED** [review]

**Pull request closed without merging**